### PR TITLE
feat: add example questions list for MAI 

### PR DESCRIPTION
**Pull request by ces-phatnguyen**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/Code-Engine-Studio/ces-semantic-release/blob/main/README.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #CU-86cu7ct5e

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: webapp/src/index.tsx [hunk 1]
```diff
@@ -1,150 +1,219 @@
...
-    return (
-        <RHSTitleContainer>
-            <IconAIContainer src={aiIcon}/>
-            {'Mai'}
-        </RHSTitleContainer>
-    );
+  return (
+    <RHSTitleContainer>
+      <IconAIContainer src={aiIcon} />
+      {"Mai"}
```
Comment (ces-sonnguyen) [inline webapp/src/index.tsx:56]: Use constant

File: server/command_hooks.go [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+	"strings"
+	"time"
+
+	"github.com/mattermost/mattermost/server/public/model"
+	"github.com/mattermost/mattermost/server/public/plugin"
+	"github.com/pkg/errors"
+)
+
+const (
+	commandTriggerMentions = "mai"
```
Comment (ces-sonnguyen) [inline server/command_hooks.go:14]: use `BotUserName`

File: webapp/src/components/rhs_questions_list/index.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
+/* eslint-disable import/no-unresolved */
+import { connect } from 'react-redux';
+
+import { getPluginChannelId } from '../../selectors';
+
+import RHSView from './rhs_view';
+
+export type stateProps = {
```
Comment (ces-sonnguyen) [inline webapp/src/components/rhs_questions_list/index.tsx:8]: StateProps

File: webapp/src/components/rhs_questions_list/rhs_view.tsx [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+  background-color: transparent;
+  width: 100%;
+`;
+
+const ButtonsContainer = styled.div`
+  display: flex;
+  gap: 5px;
+`;
+
+const defaultQuestions = [
```
Comment (ces-sonnguyen) [inline webapp/src/components/rhs_questions_list/rhs_view.tsx:61]: DefaultQuestions

**Comment by ces-sonnguyen** [comment]
Consider adding a `.prettierrc` file into web-app repo.

Here's an example
```
{
    "printWidth": 80,
    "arrowParens": "avoid",
    "semi": true,
    "tabWidth": 4
}
```

**Review by ces-sonnguyen — APPROVED** [review]

**Review by ces-vinhhuynh — APPROVED** [review]

**Pull request merged**